ui/lib/sync/addon/routes/secrets/overview.ts

```diff
@@ -6,27 +6,30 @@
 import Route from '@ember/routing/route';
 import { service } from '@ember/service';
 import { hash } from 'rsvp';
+import lazyCapabilities, { apiPath } from 'vault/macros/lazy-capabilities';
 
 import type FlagsService from 'vault/services/flags';
 import type RouterService from '@ember/routing/router-service';
 import type Store from '@ember-data/store';
 import type VersionService from 'vault/services/version';
+import type CapabilitiesModel from 'vault/models/capabilities';
 
 export default class SyncSecretsOverviewRoute extends Route {
   @service('app-router') declare readonly router: RouterService;
   @service declare readonly store: Store;
   @service declare readonly flags: FlagsService;
   @service declare readonly version: VersionService;
 
+  @lazyCapabilities(apiPath`sys/activation-flags/secrets-sync/activate`)
+  declare secretsSyncActivatePath: CapabilitiesModel;
+
   async model() {
     const isActivated = this.flags.secretsSyncIsActivated;
-    const licenseHasSecretsSync = this.version.hasSecretsSync;
-    const isHvdManaged = this.flags.isHvdManaged;
 
     return hash({
-      licenseHasSecretsSync,
-      isActivated,
-      isHvdManaged,
```

Comment:
since this data all comes from services, calling these in the relevant components felt easier to follow the logic instead of tracing back to the route model